cmake/CxxQtConfig.cmake.in

```diff
@@ -0,0 +1,10 @@
+@PACKAGE_INIT@
```

Comment:
add a SPDX header to this file or in a .license if you don't want it polluting things

Reply:
Done